src/main/kotlin/domain/MineSweeper.kt

```diff
@@ -0,0 +1,25 @@
+package domain
+
+class MineSweeper(private val board: Board, private val selector: PositionSelector) {
+    fun sweepMine(position: Position = Position()): Boolean {
+        val cell = getCell(position)
+        if (!cell.isOpen()) {
+            open(position, emptySet())
+        }
+        return !cell.isMine()
+    }
+
+    private fun open(position: Position, visited: Set<Position>) {
+        val positions = selector
+            .adjacentPositions(position)
+            .filterNot { getCell(it).isOpen() || visited.contains(it) }
+        positions
+            .forEach { getCell(it).open() }
+        positions
+            .filter { getCell(it).isBlank() }
+            .map { it to visited + setOf(it) }
+            .forEach { (position, visited) -> open(position, visited) }
```

Comment:
```suggestion
        selector
            .adjacentPositions(position)
            .asSequence()
            .filterNot { getCell(it).isOpen() || visited.contains(it) }
            .map {
                getCell(it).open()
                it
            }
            .filter { getCell(it).isBlank() }
            .toList()
            .forEach {
                open(
                    position = it,
                    visited = visited
                )
            }
```
제가 의도한 것은 이것이였어요
```kotlin
            .map {
                getCell(it).open()
                it
            }
```
map 함수의 마지막에 변환될 값을 넣어주면 되기 때문에, getCell(it).open()을 map에서 호출시켜주는 것을 의도했어요